homeassistant/components/hassio/handler.py

```diff
@@ -530,6 +531,11 @@ async def send_command(
 
         This method is a coroutine.
         """
+        url = f"http://{self._ip}{command}"
+        if url != str(URL(url)):
+            _LOGGER.error("Invalid request %s", command)
```

Comment:
Side note: Long term it would be better to pass the message to the exception and let the caller handle it, instead of logging it here.